lib/Drupal/drupal_helpers/Field.php

```diff
@@ -19,6 +19,8 @@ class Field {
    *
    * @param string $field_name
    *   Machine name of the Field.
+   *
+   * @throws \Exception
```

Comment:
@christopher-hopper please either add more information about exception or remove this tag as it is non-inforamtive